SampleApp/Sources/ExampleCustomAnimation.swift

```diff
@@ -0,0 +1,360 @@
+//
+//  StatusProgressView.swift
```

Comment:
Moved this over from SPOS since this is an actual thing we need to support, so figured may as well go right to the source for a real example...